payment/src/main/java/org/killbill/billing/payment/core/sm/OperationCallbackBase.java

```diff
@@ -58,24 +59,30 @@ protected OperationCallbackBase(final GlobalLocker locker,
     // The dispatcher may throw a TimeoutException, ExecutionException, or InterruptedException; those will be handled in specific
     // callback to eventually throw a OperationException, that will be used to drive the state machine in the right direction.
     //
-    protected <ExceptionType extends Exception> OperationResult dispatchWithAccountLockAndTimeout(final DispatcherCallback<PluginDispatcherReturnType<OperationResult>, ExceptionType> callback) throws OperationException {
+    protected <ExceptionType extends Exception> OperationResult dispatchWithAccountLockAndTimeout(final String pluginName, final DispatcherCallback<PluginDispatcherReturnType<OperationResult>, ExceptionType> callback) throws OperationException {
         final Account account = paymentStateContext.getAccount();
```

Comment:
Maybe we should rename that variable to something like `pluginNames` (since there can be multiple plugins)